infra/test/infra_test.go

```diff
@@ -56,15 +63,14 @@ func BuildAndPublish(t *testing.T) {
 
 func CreateDevEnvironmentInWorkspace(t *testing.T, terraformOptions *terraform.Options, workspaceName string) {
 	fmt.Printf("::group::Create dev environment in new workspace '%s\n'", workspaceName)
-	terraform.Init(t, terraformOptions)
```

Comment:
the call later to terraformrun was supposed to replace this but i forgot to delete this in #242